packages/typespec-ts/test/azureIntegration/payloadPageable.spec.ts

```diff
@@ -12,7 +12,8 @@ describe("Pageable Client", () => {
         });
     });
 
-    it("should get pagable Server Driven Pagination link", async () => {
+    // Not support, pending on https://github.com/Azure/autorest.typescript/issues/3022
+    it.skip("should get pagable Server Driven Pagination link", async () => {
```

Comment:
Paging metadata in RLC can't be extracted from getPageMetadata and will have follow-up issue to track this - https://github.com/Azure/autorest.typescript/issues/3022.